pdfs/src/Pdf/FromLatex.hs

```diff
@@ -146,7 +149,15 @@ produce loc ciInfo info (ResourcePath resources) =
             _ <- action
             _ <- action
 
-            LBS.readFile "texput.pdf"
+            PDFBody <$> LBS.readFile "texput.pdf"
+
+-- | Generate a PDF-certificate with the default template and return it as
+-- a bytestring.
+compileCertificate
+    :: MonadIO m
+    => Language -> CertificateIssuerInfo -> CertificateFullInfo -> m PDFBody
+compileCertificate loc ciInfo info =
+    liftIO $ produce loc ciInfo info (ResourcePath "pdfs/template")
```

Comment:
This depends on the current process directory, and it makes it painful to run the node without errors - e. g. the test for certificate building already fails in CI complaining that resources directory cannot be found. I think that 
1) this path should be configurable via config so that it could be either absolute or relative to `AppDir`
2) we should make sure that resource files are included into the deployment distribution (I'm sure they are not now)

Reply:
Going to spend another day trying to make nix digest our tests :unamused: